plugin/storage/cassandra/spanstore/writer_test.go

```diff
@@ -186,18 +190,21 @@ func TestSpanWriter(t *testing.T) {
 				serviceOperationNameQuery.On("Exec").Return(testCase.serviceOperationNameQueryError)
 				serviceOperationNameQuery.On("String").Return("select from service_operation_index")
 
+				tagsQuery := &mocks.Query{}
+				tagsQuery.On("Exec").Return(testCase.tagsQueryError)
+				tagsQuery.On("String").Return("select from tags")
+
 				durationNoOperationQuery := &mocks.Query{}
 				durationNoOperationQuery.On("Bind", matchEverything()).Return(durationNoOperationQuery)
 				durationNoOperationQuery.On("Exec").Return(testCase.durationNoOperationQueryError)
 				durationNoOperationQuery.On("String").Return("select from duration_index")
 
+				// Define expected order of queries
```

Comment:
This comment is rather misleading because one might assume that the expected ordering is enforced on asserts. I think we should remove it. 

> AssertExpectations asserts that everything specified with On and Return was in fact called as expected. Calls may have occurred in any order.

I'm not sure whether this is possible with testify/mock, but gomock provides [InOrder](https://godoc.org/github.com/golang/mock/gomock#InOrder) which does enforce ordering of expectations.